packages/service-library/src/converters/converter.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export interface Converter<DocumentType, ViewModelType> {
```

Comment:
Can we then move /factories/ under /converters/ to unify or vs

Reply:
factories are not converters rite?